app/assets/src/components/ui/controls/BulkSampleUploadTable.jsx

```diff
@@ -69,18 +78,31 @@ class BulkSampleUploadTable extends React.Component {
 
     return (
       <div className={cs.bulkSampleUploadTable}>
-        <div className={cs.detectedMsg}>
-          These files were detected and matched. Remove files you do not want to
-          upload:
-        </div>
+        {this.props.showCount ? (
```

Comment:
it's weird that the GitHub highlighting broke around here but I don't see a bug..

Reply:
Weird...